docs/examples/edb/edb_example.ipynb

```diff
@@ -463,7 +463,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.8.12"
```

Comment:
Was there a reason to revert back.?

Reply:
This is apparently just metadata that gets added to the notebook depending on who last opened/edited it. This file was originally created by Dan, but when I edited it it got reverted to my version.